tmt/steps/provision/localhost.py

```diff
@@ -1,8 +1,8 @@
 from click import echo
 
 from tmt.steps.provision.base import ProvisionBase
-
+from shlex import join
```

Comment:
we need to find something that works here for python 3.6 and 3.7. This was added in 3.8.

I currently just use as a workaround:

```
from os.path import join
```

Reply:
The `os.path.join` would not work here as it joins with slashes. We could use `shlex.quote` or even get rid of the join.

Reply:
I must've just forgotten to remove this. I've reimplemented it myself. And no it cannot be replaced with `os.path.join` AFAIK.

Reply:
Here: https://github.com/psss/tmt/pull/30/files#diff-253b78529284ad6a69cc8e0bf8d0df02R55-R61

Reply:
But that was already merged, wasn't it?